cmd/snap/cmd_info.go

```diff
@@ -469,17 +622,23 @@ func (x *infoCmd) Execute([]string) error {
 			continue
 		}
 
-		if tryDirect(w, snapName, x.Verbose, termWidth) {
-			noneOK = false
-			continue
-		}
-		remote, resInfo, _ := x.client.FindOne(snapName)
-		local, _, _ := x.client.Snap(snapName)
+		iw.arg = snapName
```

Comment:
do we ever use this value of arg?  it seems arg is only used where diskSnap != nil or am I missing something?  if that's the case maybe we should call it path perhaps?

Reply:
fair point, I'll rename it "path" and only set it if it's a disk snap